apigenerator/resources/1 - Project/1 - BaseProject/Project/src/e_Infra/b_Builders/ProxyResponseBuilder.py

```
@@ -4,6 +4,7 @@
 
 # Infra Imports #
 from src.e_Infra.CustomVariables import *
+from src.e_Infra.GlobalVariablesManager import *
```

Ruff (0.11.9)

`from src.e_Infra.GlobalVariablesManager import *` used; unable to detect undefined names

(F403)